docs/features/publishing-events.md

```diff
@@ -29,9 +33,42 @@ var @event = new NewCarRegistered(eventId, eventSubject, licensePlate);
 await eventGridPublisher.Publish(@event);
 ```
 
-Alternatively you can publish a list of events by using 
+Alternatively you can publish a list of events by using
+
 ```csharp
 await eventGridPublisher.PublishMany(events);
 ```
 
+### Resilient Publishing
+
+The `EventGridPublisherBuilder` also provides several ways to publish events in a resilient manner. Resilient meaning we support three ways to add resilience to your event publishing:
+
+1. **Exponential retry**: makes the publishing resilient by retrying a specified number of times with exponential backoff.
+
+```csharp
+EventGridPublisherBuilder.ForTopic(topicEndpoint)
+                         .UsingAuthenticationKey(endpointKey)
+                         .WithExponentialRetry(retryCount)
+                         .Build();
+```
+
+2. **Circuit broker**: makes the publishing resilient by breaking the circuit if the maximum specified number of exceptions are handled by the policy. The circuit will stay broken for a specified duration. Any attempt to execute the function while the circuit is broken will result in a `BrokenCircuitException`.
+
+```csharp
+EventGridPublisherBuilder.ForTopic(topicEndpoint)
+                         .UsingAuthenticationKey(endpointKey)
+                         .WithCircuitBreaker(exceptionsBeforeBreaking, durationOfBreak)
+                         .Build();
+```
+
```

Comment:
Line needs to be removed to fix the numbering, see https://deploy-preview-58--arcus-eventgrid-staging.netlify.com/features/publishing-events